### ISSUE TITLE
Error on run ts-node index.ts

### ISSUE THREAD

**Issue by JulioLongo** [open-wa/wa-automate-nodejs#1193]
Error described on image

![image](https://user-images.githubusercontent.com/33060329/102958162-12357b00-44bb-11eb-8423-893a968f3ca0.png)

**Comment by smashah** [comment]
@JulioLongo 

Can you go to line 524 in Client.ts and replace

```javascript
	    if(this._registeredEvListeners[funcName]) {
```

with

```javascript
	    if(this?._registeredEvListeners[funcName]) {
```

If this works for you I will create a new patch ASAP.

Thanks.

**Comment by aidilaryanto** [comment]
Same here

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.1.12](https://github.com/open-wa/wa-automate-nodejs/compare/3.1.11...3.1.12) (2020-12-23)

- Release 3.1.12 [`145a24a`](https://github.com/open-wa/wa-automate-nodejs/commit/145a24ac881a902f179d0c8f77b940cb2c5ad878)
- docs: add `crop` option to documentation #1188 [`cd0f47a`](https://github.com/open-wa/wa-automate-nodejs/commit/cd0f47ae3106c61eabcd54e18f4c972a01688b15)
- remove `done` params from documentation [`f2dbac5`](https://github.com/open-wa/wa-automate-nodejs/commit/f2dbac5962337cf70c591d8584424f63e1ef32ed)
- remove `done` params from documentation [`1b10986`](https://github.com/open-wa/wa-automate-nodejs/commit/1b109869aaa5c638317d1c3016f77d63dc59a8e0)
- fix: #1193 [`57e8eb7`](https://github.com/open-wa/wa-automate-nodejs/commit/57e8eb737d4aec3917960264060632d1244f8740)
